exporters/stdout/stdoutmetric/exporter_test.go

```diff
@@ -56,18 +55,6 @@ func testCtxErrHonored(factory func(*testing.T) func(context.Context) error) fun
 }
 
 func TestExporterHonorsContextErrors(t *testing.T) {
-	t.Run("Shutdown", testCtxErrHonored(func(t *testing.T) func(context.Context) error {
```

Comment:
Can we have tests: `TestExporterExportHonorsContextErrors`, `TestExporterForceFlushIgnoresContextErrors`, ``TestExporterShutdownIgnoresContextErrors` tests instead.

Reply:
Done - Renamed test `TestExporterHonorsContextErrors` to `TestExporterExportHonorsContextErrors`. Added tests `TestExporterForceFlushIgnoresContextErrors` and `TestExporterShutdownIgnoresContextErrors`.